Zend/zend_object_handlers.c

```diff
@@ -2545,6 +2545,7 @@ ZEND_API const zend_object_handlers std_object_handlers = {
 	zend_object_std_dtor,					/* free_obj */
 	zend_objects_destroy_object,			/* dtor_obj */
 	zend_objects_clone_obj,					/* clone_obj */
+	zend_objects_clone_obj_with,					/* clone_obj_with */
```

Comment:
Nit: Inconsistent comment indentation for default spacing (1 tab -> 4 spaces).

Reply:
Heh. I used to use two but switched back to 4 basically for this reason. Optimally, we'd use spaces for anything other than the spacing after a `\n`.